### PR TITLE
Polish UI for Apply Diff View

### PR DESCRIPTION
**Pull request by Patriot99**

### CODE REVIEW

File: src/NexusMods.App.UI/Resources/Language.pl.resx [hunk 1]
```diff
@@ -181,7 +181,7 @@
 		<value>Wstrzymaj wszystko</value>
 	</data>
 	<data name="InProgressTitleTextBlock" xml:space="preserve">
-		<value>Pobierane</value>
+		<value>Pobrane</value>
```
Comment (Sewer56) [inline src/NexusMods.App.UI/Resources/Language.pl.resx:184]: Doesn't `Pobrane` mean `Downloaded`?

The 'accurate' translation here is probably `W Trakcie Pobierania` or rather `W Trakcie`, but I imagine that may be long or unfriendly.

File: src/NexusMods.App.UI/Resources/Language.pl.resx [hunk 2]
```diff
@@ -139,10 +139,10 @@
 		<value>Dodaj grę</value>
 	</data>
 	<data name="DownloadsView__DOWNLOADS" xml:space="preserve">
-		<value>POBRANE</value>
+		<value>POBIERANIE</value>
```
Comment (Sewer56) [inline src/NexusMods.App.UI/Resources/Language.pl.resx:142]: This should probably still say `POBRANE` as `POBERIANIE` is DOWNLOAD(ING), but this string is unused currently it seems.

File: src/NexusMods.App.UI/Resources/Language.pl.resx (hunk 1 above)
Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:184]: In the UI, it looks like a "Downloads" folder (a folder that contains downloaded files), just like in Windows Explorer, so the translation in this case is correct.

Screenshot_20240411-004817~2

Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:184]: Optionally, I can restore the previous name  `Pobierane`. This is a shorter alternative to `W trakcie pobierania`.

Comment (Patriot99) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:184]: Or `Pobieralnia`. Tab where you can download files.

Comment (Sewer56) [reply src/NexusMods.App.UI/Resources/Language.pl.resx:184]: `Pobieralnia` sounds great, actually

**Review by Sewer56 — APPROVED** [review]

**Pull request merged**